lib/packs/rails/railtie.rb

```diff
@@ -11,7 +11,19 @@ class Railtie < ::Rails::Railtie
         # hook into packs-rails via ActiveSupport hooks.
         ActiveSupport.run_load_hooks(:packs_rails, Packs)
       end
-      
+
+      config.after_initialize do |app|
+        if defined?(Spring)
+          puts "Hello Db Blasters!"
+          Packs::Specification::Configuration.fetch.pack_paths.each do |dir|
+            puts "#{dir}"
+            Dir["#{dir}/package.yml"].each { |f|
```

Comment:
why would you do a glob here (`Dir[]`)? Isn't `dir` the exact path? In which case you'll want `File.join(dir, "package.yml")` or something like that

But I'm probably misunderstanding `pack_paths`...

Reply:
Great question!

Example `pack_paths`:
- packs/*
- packs/*/*

Reply:
ah, makes sense!

Reply:
I think `pack_paths` is easy to misunderstand; it's more like `pack_path_specs` or `pack_path_globs`, but that's out of scope here